app/services/planning_application_search.rb

```diff
@@ -7,7 +7,10 @@ class PlanningApplicationSearch
   STATUSES = %w[not_started invalidated in_assessment awaiting_determination to_be_reviewed].freeze
   REVIEWER_STATUSES = %w[awaiting_determination to_be_reviewed].freeze
 
+  APPLICATION_TYPES = %w[lawfulness_certificate prior_approval].freeze
+
   attribute :view, :enum, values: %w[all mine], default: "mine"
+  attribute :application_type, :list
```

Comment:
how about adding the default here?
```
attribute :application_type, :list, default: APPLICATION_TYPES
```

also after merging the PR #1116 we can do `ApplicationType::NAME_ORDER`

Reply:
Happy for you to add this change in your PR 👍